lib/salestation/web/ip_address.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Salestation
+  class Web < Module
+    module IPAddress
+      def self.extract(request)
+        if request['HTTP_X_FORWARDED_FOR'].nil?
+          request['REMOTE_ADDR']
+        else
+          request['HTTP_X_FORWARDED_FOR'].split(',').first
+        end
```

Comment:
NIT: I think it's easier to read if instead of checking for the inexistence of A then use B, you would check the existence of A then use A.

Since there are`if` and `else` it's ok to leave it as is.